.github/actions/start-docker-daemon/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'start_docker_daemon'
+description: 'start_docker_daemon'
+inputs:
+  container-id:  # id of input
+    description: 'The running container id'
+    required: true
+runs:
+  using: "composite"
+  steps:
+  - id: start-docker-daemon
+    run: |
+      docker exec -d ${{ inputs.container-id }} bash -c '
+      while [ ! -f /var/run/docker.sock ] ;
+      do
+        dockerd -b docker0 --storage-driver=vfs;
```

Comment:
why not run in background mode?

Reply:
If I add `&`, then the shell script will get the error as follow. I found delete `&` is ok because `docker exec -d` can achieve a similar effect.
```
# docker exec -t test bash -c 'while [ ! -f /var/run/docker.sock ] ; do dockerd -b docker0 --storage-driver=vfs &; exit -1; done'
bash: -c: line 0: syntax error near unexpected token `;'
bash: -c: line 0: `while [ ! -f /var/run/docker.sock ] ; do dockerd -b docker0 --storage-driver=vfs &; exit -1; done'

# docker exec -t test bash -c 'while [ ! -f /var/run/docker.sock ] ; do dockerd -b docker0 --storage-driver=vfs; exit -1; done'
......
INFO[2021-06-23T15:46:07.964155276Z] API listen on /var/run/docker.sock
```

Reply:
The test result is in [link](https://github.com/haosanzi/inclavare-containers/runs/2896682425?check_suite_focus=true)